wrappers/python/examples/wnt/README.md

```diff
@@ -0,0 +1,73 @@
+# WNT API Examples
+
+This folder contains the Wirepas Network Tool API Examples. These examples show how to interact with
+the server using authentication -, metadata - and real time situation connections.
+
+Please not that these examples should not be taken as production quality, but more
+as the examples about the message flow and content.
+
+## Requirements
+
+Before running the examples please ensure that the 
+[wirepas_messaging](https://github.com/wirepas/backend-apis/tree/master/wrappers/python)
+package is installed.
+
+## WNT backend settings
+
+For running the examples, WNT backend connection parameters needs be given by creating settings.yml
+file to the same folder from where the examples are run.
+
+### Example settings file
+```shell
+username: "username"
+password: "password"
+hostname: "wntbackend.mydomain.com"
+```
+
+## Examples
+
+  - [applicationconfiguration.py](https://github.com/wirepas/backend-apis/tree/master/wrappers/python/examples/wnt/applicationconfiguration.py)
+    - Wirepas mesh application configuration and diagnostics interval setting *
+
+  - [authentication.py](https://github.com/wirepas/backend-apis/tree/master/wrappers/python/examples/wnt/authentication.py)
+    - Authentication and user querying, creation, updating and deletion.
+
+  - [building.py](https://github.com/wirepas/backend-apis/tree/master/wrappers/python/examples/wnt/building.py)
+    - Building querying, creation, updating and deletion
+
+  - [componentsinformation.py](https://github.com/wirepas/backend-apis/tree/master/wrappers/python/examples/wnt/componentsinformation.py)
```

Comment:
Could you used named links in order to keep the document more clean?

`[text][link]`

At the end of the document:

`[link]: https://`

Reply:
Done